contents/handbook/engineering/incidents.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Handling an incident
+sidebar: Handbook
+showTitle: true
+---
+
+Incidents are going to happen.
+
+## When to raise an incident
+
+**When in doubt, raise an incident.** We'd much rather have declared an incident which turned out not to be an incident. Many incidents take too long to get called, or are missed completely because someone didn't ring the alarm when they had a suspicion something was wrong.
+
+To declare an incident, type `/incident` in the #incidents channel. This will create a new channel and send updates.
+
+Anyone can declare an incident, including non-engineers. If in doubt, check with your nearest engineer.
+
+Some things that should definitely be an incident
+- `app.posthog.com` being completely unavailable (not just for you)
+- No insights can be created
```

Comment:
Instead of explicitly calling what should be an incident, I would probably rephrase it as "every core functionality of the app works as designed"

Reply:
I'm calling out a couple of things specifically as examples as people may have a different definition of 'core functionality'